### ISSUE TITLE
Is the tag event broadcast implemented

### ISSUE THREAD

**Issue by gugaoliveira** [yagmure15/zebra_rfid_reader_sdk#1]
Hello, thanks for the excellent work, The tag is reported by Android in kotlin library, I couldn't catch tag read event in Flutter sdk, it is complete?

**Comment by JarrodLodge** [comment]
I got it working by changing this in zebra_rfid_reader/android/src/main/kotlin/com/borda/zebra_rfid_reader_sdk/RfidEventHandler.kt

override fun eventReadNotify(e: RfidReadEvents) {
        val myTags: Array<TagData> = reader.Actions.getReadTags(100)
        var data: MutableList<String> = mutableListOf<String>()
        for (tag in myTags) {
            Log.d(LOG_TAG, "Tag ID " + tag.tagID)
            data.add(ReaderTagData(tag.tagID, tag.peakRSSI.toInt()).toJson())
        }
        emit(Gson().toJson(data))
    }
    
    I don't know Kotlin, I just asked chatGPT, so not sure if this is the best way.

**Comment by gugaoliveira** [comment]
Yes, but I think the best way is to create another event channel, it is mixing the Control events with the tag events. But thank you for the help

**Comment by JarrodLodge** [comment]
Yes I think you're right. That would make it cleaner.

**Comment by gugaoliveira** [comment]
@JarrodLodge Have you wrote tags with the zebra SDK with flutter?

**Comment by JarrodLodge** [comment]
> @JarrodLodge Have you wrote tags with the zebra SDK with flutter? 

No I haven't tried yet. We only need to read them for our application.

**Comment by serapion123321** [comment]
Still have an issue here. readTags not defined in version 0.0.2
<img width="571" alt="Screenshot 2024-07-29 at 11 02 05" src="https://github.com/user-attachments/assets/f7c492fe-c6fd-4e12-a09e-0abfb3f340ff">
<img width="890" alt="Screenshot 2024-07-29 at 11 02 22" src="https://github.com/user-attachments/assets/b06c5c29-3db1-48bd-992b-bf96b5f97783">
<img width="937" alt="Screenshot 2024-07-29 at 11 04 07" src="https://github.com/user-attachments/assets/cafc91af-96c7-482e-a3a3-47957853d5c2">

**Comment by yagmure15** [comment]
hi @serapion123321, did you read readme?
https://github.com/yagmure15/zebra_rfid_reader_sdk#notes. please check it.

**Comment by serapion123321** [comment]
> hi @serapion123321, did you read readme? https://github.com/yagmure15/zebra_rfid_reader_sdk#notes. please check it.

@yagmure15 ok Thanks, solved
